SinaiStCatherine/SinaiEt004.xml

```diff
@@ -0,0 +1,894 @@
+                                    <item xml:id="q6" n="5">
+                                        <dim unit="leaf">8</dim>
+                                        <locus from="39r" to="46v"/>
+                                    </item>
+                                    
+                                    <item xml:id="q7" n="6">
+                                        <dim unit="leaf">8</dim>
+                                        <locus from="47r" to="54v"/>                                      
+                                    </item>
+                                    
+                                    <item xml:id="q8" n="7">
+                                        <dim unit="leaf">8</dim>
+                                        <locus from="55r" to="62v"/>
+                                    </item>
+                                    
+                                    <item xml:id="q9" n="8">
+                                        <dim unit="leaf">8</dim>
+                                        <locus from="63r" to="68v"/>
+                                    </item>
+                                    
+                                    <item xml:id="q10" n="9">
+                                        <dim unit="leaf">8</dim>
+                                        <locus from="69r" to="76v"/>
+                                    </item>
+                                    
+                                    <item xml:id="q11" n="10">
+                                        <dim unit="leaf">8</dim>
+                                        <locus from="77r" to="84v"/>
+                                    </item>
+                                    
+                                    <item xml:id="q12" n="11">
+                                        <dim unit="leaf">8</dim>
+                                        <locus from="85r" to="92v"/>                                        
+                                    </item>
+                                    
+                                    <item xml:id="q13" n="12">
+                                        <dim unit="leaf">10</dim>
+                                        <locus from="93r" to="102v"/>
+                                    </item>
+                                    
+                                    <item xml:id="q14" n="13">
+                                        <dim unit="leaf">12</dim>
+                                        <locus from="103r" to="114v"/>
+                                      <note>Structure uncertain</note>
+                                    </item>
+                                    
+                                    <item xml:id="q15" n="14">
+                                        <dim unit="leaf">11</dim>
+                                        <locus from="115r" to="125v"/>
+                                      <note>Structure uncertain</note>
```

Comment:
if no info on s.l. is given the script for "drawing" quires does not work.

Reply:
Yes, but I cannot decipher the structure of this part on photos.